common/app/model/dotcomrendering/pageElements/EditionsCrosswordRenderingDataModel.scala

```diff
@@ -1,33 +1,71 @@
 package model.dotcomrendering.pageElements
 
-import com.gu.contentapi.client.model.v1.Crossword
-import com.gu.contentapi.json.CirceEncoders._
-import io.circe.syntax._
-import implicits.Dates.CapiRichDateTime
-import model.dotcomrendering.DotcomRenderingUtils
-import play.api.libs.json.{JsObject, Json, JsValue}
+import com.gu.contentapi.client.model.v1.{CrosswordCreator, CrosswordDimensions, CrosswordEntry}
+import play.api.libs.json._
+
+import scala.collection.Seq
+
+case class CrosswordPosition(x: Int, y: Int)
+
+object CrosswordPosition {
+  implicit val jsonWrites: OWrites[CrosswordPosition] = Json.writes[CrosswordPosition]
+}
+
+case class EditionsCrosswordEntry(
+    id: String,
+    number: Int,
+    humanNumber: String,
+    clue: String,
+    direction: String,
+    length: Int,
+    group: Seq[String],
+    position: CrosswordPosition,
+    separatorLocations: Option[Map[String, Seq[Int]]],
+    solution: Option[String],
+)
+
+object EditionsCrosswordEntry {
+  implicit val jsonWrites: OWrites[EditionsCrosswordEntry] = Json.writes[EditionsCrosswordEntry]
+
+  def fromCrosswordEntry(entry: CrosswordEntry, shipSolutions: Boolean): EditionsCrosswordEntry = {
+    EditionsCrosswordEntry(
+      id = entry.id,
+      number = entry.number.getOrElse(0),
+      humanNumber = entry.humanNumber.getOrElse(""),
+      clue = entry.clue.getOrElse(""),
+      direction = entry.direction.getOrElse(""),
+      length = entry.length.getOrElse(0),
+      group = entry.group.getOrElse(Seq.empty),
```

Comment:
We could do this so that we keep the sequences in this file immutable, and therefore don't need to import and use the mutable sequence since we're not mutating anything?

```suggestion
      group = entry.group.map(_.toSeq).getOrElse(Seq.empty),
```